plugins/vpp/ifplugin/vppcalls/vpp1904/ipip_vppcalls.go

```diff
@@ -0,0 +1,31 @@
+//  Copyright (c) 2020 Cisco and/or its affiliates.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at:
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package vpp1904
+
+import (
+	"fmt"
+
+	interfaces "go.ligato.io/vpp-agent/v3/proto/ligato/vpp/interfaces"
+)
+
+// AddIpipTunnel adds new IPIP tunnel interface.
+func (h *InterfaceVppHandler) AddIpipTunnel(ifName string, vrf uint32, ipipLink *interfaces.IPIPLink) (uint32, error) {
+	return 0, fmt.Errorf("IPIP interface unsupported in VPP 1904")
```

Comment:
Define this error in `vppcalls` package as exported so it can be unwrapped with [`errors.Is(...)`](https://pkg.go.dev/errors?tab=doc#Is). And then register the error in the ifplugin's `init()` as non-retryable error to KVScheduler:

```go
func init() {
    // ...
	kvscheduler.AddNonRetryableError(vppcalls.ErrIPIPUnsupported)
}
```

This will prevent KVScheduler from retrying the transaction. 

Then you can return it here wrapped with specific version as:
```go
   return fmt.Errorf("%w in VPP 19.04", vppcalls.ErrIPIPUnsupported) // %w wraps the error
```